docs/data/external-oasis-repos.md

```diff
@@ -63,94 +82,171 @@ root@host # chkconfig httpd on
+        Please add a new CVMFS repository to OASIS for VO %RED%voname%ENDCOLOR% using the URL 
+            http://%RED%fully.qualified.domain%ENDCOLOR%:8000/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%
+        by doing step #3 at
+            https://opensciencegrid.github.io/docs/data/external-oasis-repos
+        The VO responsible manager will be %RED%OASIS Manager Name%ENDCOLOR%.
 
-6.  The GOC representative then adds the URL in OIM under OASIS Repo URLs for the VO. The repository will then be added to the GOC stratum 0 within 15 minutes.  Within an hour after step 8 is completed, the repository should also be automatically added to the GOC and FNAL stratum 1s.
+    Replace the %RED%red%ENDCOLOR% items with the appropriate values.
 
-7.  If domain.name is a new domain that has not been distributed before, the GOC representative next places a copy of the domain.name.pub public key from domain.name into /srv/etc/keys on both oasis-replica and oasis-replica-itb. If the GOC representative does not have that key, he or she can ask the repository service representative in the ticket how to get it. In addition, in order to support cvmfs client versions 2.2.X (both older and newer clients do not need it), a symbolic link of `domain.name`.conf has to be made in /cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d pointing to default.conf. This symbolic link has to be created on the oasis-itb machine's copy of the config-osg.opensciencegrid.org repository and then copied to production with the `copy_config_osg` command on the oasis machine.
+3.  (**OSG internal step**)
+    OSG Operations ensures that the repository administrator is valid for the VO. This can be done by (a) OSG
+    already having a relationship with the person or (b) the GOC representative contacting the VO manager to find out.
+    OSG Operations should review the URL to verify it is appropriate for the VO.  This typically takes one business day.
 
-8.  If the repository name matches `*.opensciencegrid.org` or `*.osgstorage.org`, the GOC representative responds in the ticket to ask that step \#8 be done; all other repositories (such as `*.egi.eu`) skip this step. The repository service administrator next executes the following commands (replacing repo.opensciencegrid.org with repo.osgstorage.org when needed):
+4.  (**OSG internal step**)
+    The OSG representative adds the repository URL in OIM under the VO's OASIS repository URLs. This should cause
+    the repository's configuration to be added to the GOC Stratum-0 within 15 minutes.  Within an hour step 8 is
+    completed, the repository should also be automatically added to the GOC and FNAL stratum 1s.
```

Comment:
Wait, now I see it is step 6 and not step 8, and anyway step 6 mentions what happens then, so just delete the sentence beginning "With an hour".